test/Vault.spec.ts

```diff
@@ -1079,9 +1253,52 @@ describe("Vault", () => {
           .claimYield("0x0000000000000000000000000000000000000000")
       ).to.be.revertedWith("Vault: destination address is 0x");
     });
+
+    it("updates shares after claim", async () => {
```

Comment:
You can change this one to `updates the shares` only.

There's some overlap between this and https://github.com/sandclock-org/solidity_contracts/pull/47/files#diff-f57911af7b18dc30725cc32a5d391e3faf71757500a45687928860c6a1e2c483R1205 so you may want to remove the checks from the other one

Reply:
I think there is no overlap.

https://github.com/sandclock-org/solidity_contracts/pull/47/files#diff-f57911af7b18dc30725cc32a5d391e3faf71757500a45687928860c6a1e2c483R1205

This line is to check claimable yield's share,

and current line is to check totalShares of Claimer